armi/resources/nuclides.dat

```diff
@@ -0,0 +1,4664 @@
+# ---------------------------------------------------------------------
+# NUCLIDE DATA
+# ---------------------------------------------------------------------
+# This file contains a set of 4614 nuclides and their associated metadata
+# of atomic number (Z), number of neutrons (N), mass number (A), stability
+# state number (S), Mass (amu), natural abundance (%), half-life (sec), and
+# the spontaneous fission neutron yield (nu-bar).
+# 
+# The stability state is either a zero (0), representing the ground state, or
+# higher (e.g., 1, 2, 3, etc.) representing excited/metastable states. In
+# most instances, the ground state represents the most commonly produced isotope
+# from fission or activation, but there are some known exceptions. For example,
+# AM242M is a more common nuclide produced from activation than the ground state
+# of AM242 and TA180M is the naturally occurring isomer of TA180.
+#
+# The mass data is acquired from (1) by parsing the FRDM database provided
+# with RIPL-3. The masses in the FRDM database are given in Mass Excess (MeV)
+# and are coverted to amu by applying the following equation:
+#
+#                  MAMU = A + ME / C
+# 
+#   Where:
+# 
+#    MAMU is the mass in units of amu,
+#       A is the mass number,
+#      ME is the mass excess in units of MeV, and
+#       C is the atomic mass to energy equivalency 
+#         constant in units of MeV/amu -- 931.49410242 MeV/amu.
+# 
+#  Note: If the nuclide mass has a "*" denoted at the end of the value then this
+#        is still taken from the FRDM database, but the value is calculated rather
+#        than experimentally measured.
+#
+# The nuclide natural abundance data is acquired from (1) by parsing the "abundance"
+# data file. The original source of this data is from the Brookhaven Nuclear Wallet Cards (2).       
+#
+# The half-life data for each nuclide is provided as a direct output from reading and 
+# processing the RIPL-3 "levels" data. A half-life provided as `inf` represents a stable 
+# nuclide and internally this is processed as "infinity" using the Python numpy package. 
+# Note that for some nuclides where the half-life is not known (i.e., TI48M), 
+# the half-life is set to a fixed value of 11.111 seconds. This value is set arbitrarily, 
+# but aligns with an assumption made in TerraPower's internal depletion solver. In almost 
+# all instances this assumption will not be consequential for practical reactor physics 
+# applications, but this should be verified on a case-by-case basis. 
+#
+# Sources: (1) Reference Input Parameter Library (RIPL-3) - 
+#              https://www-nds.iaea.org/RIPL-3/
+#          (2) J.K. Tuli, Nuclear Wallet Cards (NNDC, Brookhaven National Laboratory), 2005.
+# ---------------------------------------------------------------------
+Z	N	A	S	El	  Mass, amu	        Abundance, %      Half-life (sec)   Spontaneous Fission Yield (nu-bar)
```

Comment:
The abundances in this file appear to be fractions, not percent, as is currently indicated in the header line.

Reply:
Thanks! Originally I had percents but changes to fractions so I'll update.